packages/cli/src/register-command.ts

```diff
@@ -17,8 +17,8 @@ export interface ParamDetails {
   prompt?: string;
   promptType?: 'confirm' | 'list' | 'input';
   choices?: readonly Choice[];
-  preselect?: string;
-  validationError?: (value: string | boolean) => string | undefined;
+  preselect?: string | boolean;
+  validationError?: (value: string | boolean) => string;
```

Comment:
Why did you remove `undefined`?

Reply:
This was likely a leftover of my exploration with `validationError`. Thanks for spotting it, and sorry!